cmd/troubleshoot/builder.go

```diff
@@ -175,46 +175,3 @@ func createTransport(ctx context.Context, apiReader client.Reader, dk *dynakube.
 
 	return registry.PrepareTransportForDynaKube(ctx, apiReader, transport, dk)
 }
-
-func getDynakubes(ctx context.Context, log logd.Logger, apiReader client.Reader, namespaceName string, dynakubeName string) ([]dynakube.DynaKube, error) {
```

Comment:
why remove it and not just use it instead of https://github.com/Dynatrace/dynatrace-operator/pull/3669/files#diff-f9bf3e2603e3edfceb8d8a0bee9129d49ce3ea3655001f89ee0f7f0f059ef0b3R108-R113 ?
```go
	dynakubes := &dynakube.DynaKubeList{}
	err = apiReader.List(ctx, dynakubes, &client.ListOptions{Namespace: namespaceName})
	if checkCRD(log, err) != nil {
		return
	}
```

this "convoluted" logic's purpose was to give a more descriptive log for the "troubleshooting" experience.

also now the following is also deadcode 😬 
```
cmd.PersistentFlags().StringVarP(&dynakubeFlagValue, dynakubeFlagName, dynakubeFlagShorthand, "", "Specify a different Dynakube name.")
```

you can no longer (since `getDynakubes` is not used ) just run the troubleshoot script against only 1 dk:
- is this a problem? 🤷

Reply:
sorry I don't understand you comment/